### ISSUE TITLE
Dropbox link broken

### ISSUE THREAD

**Issue by jrfinkbeiner** [PIX2NVS/NVS2Graph#18]
Hi All,

thanks for the great dataset!
It seems like the [dropbox link](https://www.dropbox.com/sh/ibq0jsicatn7l6r/AACNrNELV56rs1YInMWUs9CAa?dl=0) that is provided in the README is not working anymore because the corresponding file was deleted. 

Is there a plan to host the dataset via dropbox again in the same format, or will it only stay available via the format of the provided google [drive link](https://drive.google.com/drive/folders/1tK5OY3pkjppYwAnLF8bnxGdaFbYEA8iY?usp=sharing)?

Thanks!
Jan

**Comment by 2561681244** [comment]
Hi All,

thanks for the great dataset!
It seems like the [dropbox link](https://www.dropbox.com/sh/ibq0jsicatn7l6r/AACNrNELV56rs1YInMWUs9CAa?dl=0) that is provided in the README is not working anymore because the corresponding file was deleted.

Is there a plan to host the dataset via dropbox again in the same format, or will it only stay available via the format of the provided google [drive link](https://drive.google.com/drive/folders/1tK5OY3pkjppYwAnLF8bnxGdaFbYEA8iY?usp=sharing)?

Thanks!
Wang

**Comment by fangwei123456** [comment]
Hi guys, you can download this dataset from the mirror:

https://github.com/fangwei123456/spikingjelly/blob/f32429ad5d83b22fad4a6e9a02fca171752a47e6/spikingjelly/datasets/asl_dvs.py#L40

```
The ICCV2019_DVS_dataset.zip is packed by dropbox. We find that the MD5 of this zip file can change. So, MD5 check will not be used for this ASL-DVS dataset.
Update: The Dropbox link is expired now. You can download this dataset from the OpenI mirror manually by the following commands:
----------
pip install openi
openi dataset download OpenI/spikingjelly ASLDVS.zip  --cluster NPU --save_path .
----------
Then you can extract "ASLDVS.zip" and get "ICCV2019_DVS_dataset.zip".
```

**Comment by YmShan** [comment]
> Hi guys, you can download this dataset from the mirror:
> 
> https://github.com/fangwei123456/spikingjelly/blob/f32429ad5d83b22fad4a6e9a02fca171752a47e6/spikingjelly/datasets/asl_dvs.py#L40
> 
> ```
> The ICCV2019_DVS_dataset.zip is packed by dropbox. We find that the MD5 of this zip file can change. So, MD5 check will not be used for this ASL-DVS dataset.
> Update: The Dropbox link is expired now. You can download this dataset from the OpenI mirror manually by the following commands:
> ----------
> pip install openi
> openi dataset download OpenI/spikingjelly ASLDVS.zip  --cluster NPU --save_path .
> ----------
> Then you can extract "ASLDVS.zip" and get "ICCV2019_DVS_dataset.zip".
> ```

Yes, this method allows you to download a compressed file named ASLDVS.zip, which when unzipped provides ICCV2019_DVS_dataset.zip.
Additionally, the author has provided a Google Drive download option. However, due to size limitations, the results downloaded from Google Drive require manual organization. I highly recommend using the download method provided by Wei Fang. I am extremely grateful to Wei Fang for their significant contribution to this project.

**Comment by 2561681244** [comment]
Than you very much!



---Original---
From: "Yimeng ***@***.***&gt;
Date: Tue, Apr 29, 2025 20:25 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [PIX2NVS/NVS2Graph] Dropbox link broken (Issue #18)


 YmShan left a comment (PIX2NVS/NVS2Graph#18)
  
Hi guys, you can download this dataset from the mirror:
 
https://github.com/fangwei123456/spikingjelly/blob/f32429ad5d83b22fad4a6e9a02fca171752a47e6/spikingjelly/datasets/asl_dvs.py#L40
 The ICCV2019_DVS_dataset.zip is packed by dropbox. We find that the MD5 of this zip file can change. So, MD5 check will not be used for this ASL-DVS dataset. Update: The Dropbox link is expired now. You can download this dataset from the OpenI mirror manually by the following commands: ---------- pip install openi openi dataset download OpenI/spikingjelly ASLDVS.zip  --cluster NPU --save_path . ---------- Then you can extract "ASLDVS.zip" and get "ICCV2019_DVS_dataset.zip".   
Yes, this method allows you to download a compressed file named ASLDVS.zip, which when unzipped provides ICCV2019_DVS_dataset.zip.
 Additionally, the author has provided a Google Drive download option. However, due to size limitations, the results downloaded from Google Drive require manual organization. I highly recommend using the download method provided by Wei Fang. I am extremely grateful to Wei Fang for their significant contribution to this project.

**Comment by skxooo** [comment]
> Than you very much!
> […](#)

Hello, I tried using the command line provided by Fangwei, but I still couldn't download the dataset. Could you please help me?

<img width="1959" height="105" alt="Image" src="https://github.com/user-attachments/assets/6224133b-a163-4fa3-98fd-34936a5bcf9b" />

**Comment by skxooo** [comment]
> > Than you very much!
> > […](#)
> 
> Hello, I tried using the command line provided by Fangwei, but I still couldn't download the dataset. Could you please help me?
> 
> <img alt="Image" width="1959" height="105" src="https://private-user-images.githubusercontent.com/49916824/490867258-6224133b-a163-4fa3-98fd-34936a5bcf9b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgxNjUwODAsIm5iZiI6MTc1ODE2NDc4MCwicGF0aCI6Ii80OTkxNjgyNC80OTA4NjcyNTgtNjIyNDEzM2ItYTE2My00ZmEzLTk4ZmQtMzQ5MzZhNWJjZjliLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTE4VDAzMDYyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2NjViNmFjMGE3M2M5NmQ0MzZlYmM4ODA2MGMxYWI3NzJmMjgwYjE4YWUwMDQ0ZjI0MzgwNDc0ZmExZDM3NDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mMMO92WyvHWLexygJdOkQMeJ7nKis4pEiNEU9rkc84E">

This command line works:

```bash
openi dataset download OpenI/ASLDVS --local_dir ./ASLDVS --max_workers 10
